tests/testStreamOpen.cpp

```diff
@@ -331,6 +331,17 @@ TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsErrorBeforeLollipop){
     }
 }
 
+TEST_F(StreamOpenOutput, PlaybackFormatFloatReturnsFloatBeforeLollipopWithFormatConversionAllowed){
+    if (getSdkVersion() < __ANDROID_API_L__){
+        mBuilder.setDirection(Direction::Output);
```

Comment:
Should we also test for Input?

Reply:
IIRC float input is only supported on Marshmallow (API 23+), so that would require a different test.

Reply:
Right. I can add a Quirk for that later.